x-pack/solutions/security/plugins/security_solution/public/detections/components/alerts_table/alerts_sub_grouping.tsx

```diff
@@ -71,6 +71,20 @@ interface OwnProps {
   signalIndexName: string | undefined;
   tableId: TableIdLiteral;
   to: string;
+
+  /**
+   * By default, multi-value fields will be transformed into a string
```

Comment:
I would slightly rephrase `By default,` into `If you're not using this property,`